### ISSUE TITLE
Add ability to have custom basemaps

### ISSUE THREAD

**Issue by npeihl** [Esri/dojo-bootstrap-map-js#89]
Looks like there's already a TODO for this [here](https://github.com/Esri/dojo-bootstrap-map-js/blob/master/src/app/config.js#L80-L81). We prefer to use our own custom basemaps rather than Esri's (nah-nah). 

If I can implement, I'll submit a PR. No promises though.

**Comment by tomwayson** [comment]
Yeah, there's a TODO, but no plans (on my part) to actually do it.

I am committed to #78, which is related, but can be done independently.

If you end up doing it, we'll gladly take the the PR!
